meta/Main.hs

```diff
@@ -112,16 +112,11 @@ meta (MetaOptions {..}) = do
           PreludeDefault typ ->
             Expr.Embed
               ( importFile ( relativeTo localDest ( defaultFile typ ) ) )
-          PreludeConstructorsLicense ->
-            Expr.Constructors ( Expr.Var "types" `Expr.Field` "License" )
-          PreludeConstructorsRepoKind ->
-            Expr.Constructors ( Expr.Var "types" `Expr.Field` "RepoKind" )
-          PreludeConstructorsScope ->
-            Expr.Constructors ( Expr.Var "types" `Expr.Field` "Scope" )
           PreludeV ->
             Expr.Embed
               ( importFile ( relativeTo localDest "./Version/v.dhall" ) )
-
+          other -> resolveVar other
```

Comment:
This resort to `resolveVar` could be problematic in a future?

Reply:
Good thought.

If we wanted to be safer, maybe we could have `data Reference = Prelude PreludeReference | Type TypeReference`. This pattern match here doesn't care about `types.dhall`, but it can't use `prelude.dhall`.

Reply:
@quasicomputational  yeah, i'll try to address that asap

Reply:
@quasicomputational 
* I've separated completely prelude from types in reference resolution
* I've extracted out all refs to "prelude" to dedicated `resolvePreludeVar` function (including SPDX and version ranges)
* The `Default` type alias accepts `PreludeReference`
* So i could remove entirely the type `Reference` and the generic `resolveVar` function